pkg/codegen/internal/test/testdata/aws-webserver.pp.go

```diff
@@ -25,6 +25,7 @@ func main() {
 		if err != nil {
 			return err
 		}
+		opt0 := true
```

Comment:
Out of scope for this PR, but it might be nice to have helper methods to generate references for optional types without requiring a temporary variable. I believe Kubernetes client-go has something like this.

Reply:
Yeah, I'm in this camp as well. My understanding was that it is more idiomatic in go to spell stuff like this out even if repetitive. This was the general feedback I got when suggesting we add helpers to the go SDK to translate prompt arrays to inputty arrays. @pgavlin can comment more here.

Reply:
It is less idiomatic to export simple, type-specific helper methods. I'm ambivalent about adding unexported helper methods to the generated code.